app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.player import Player
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
Do not use list comprehension inside of the `sum`, use generator expression as mentioned in the [checklist](https://github.com/voronvasia/py-elves-and-dwarves/blob/develop/checklist.md#code-efficiency).